packages/wxt/src/core/utils/building/resolve-config.ts

```diff
@@ -253,6 +253,8 @@ function resolveZipConfig(
       // Tests
       '**/__tests__/**',
       '**/*.+(test|spec).?(c|m)+(j|t)s?(x)',
+      // Output directory
+      `${mergedConfig.outDir ?? '.output'}/**`,
```

Comment:
Pass the `outBaseDir` into this function like the `root` instead of recomputing it here.